inference-engine/src/gna_plugin/gna_plugin.cpp

```diff
@@ -765,8 +760,10 @@ void GNAPlugin::LoadNetwork(CNNNetwork & _network) {
     int passIdx = 0;
     auto run_passes = [&] (const CNNNetwork& network, bool runBeforeCopy, bool lowPrecision) {
         auto passes = make_shared<PassManager>(PassManagerSettings{runBeforeCopy, lowPrecision}, network);
-        passes->registerPass<RemoveConstPass>();
-        passes->registerPass<UnrollTIPass>();
+        if (!isNgraphPassesUsed) {
+            passes->registerPass<RemoveConstPass>();
+            passes->registerPass<UnrollTIPass>();
+        }
         passes->registerPass<RemoveConstPass>();
```

Comment:
Why we can't move it under `if` and use` ConstantFolding` pass where it is necessary for the ngraph function?